src/examples/accordions/stepper/StepperDefault.tsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Stepper } from '@zendeskgarden/react-accordions';
+import { Button } from '@zendeskgarden/react-buttons';
+
+const StyledButtons = styled.div`
+  margin-top: ${p => p.theme.space.sm};
```

Comment:
Looks like https://github.com/zendeskgarden/website/pull/166#discussion_r483304391 is still a problem.

Reply:
Good spot, missed adding the padding here.